mobile/src/main/java/rs/readahead/washington/mobile/mvp/presenter/MediaImportPresenter.java

```diff
@@ -28,7 +28,7 @@ public MediaImportPresenter(IMediaImportPresenterContract.IView view) {
 
     @Override
     public void importImage(final Uri uri) {
-        disposables.add(Observable.fromCallable(() -> MediaFileHandler.importPhotoUri(view.getContext(), uri,null))
+        disposables.add(Observable.fromCallable(() -> MediaFileHandler.importPhotoUri(view.getContext(), uri,null).blockingGet())
```

Comment:
Have you been able to test if this affected the behavior of the app?

Reply:
yes, i have imported a photo and it didn't affect anything